packages/kit/src/views/AccountManagerStacks/pages/AccountSelectorStack/WalletDetails/WalletDetailsHeader/index.tsx

```
@@ -56,69 +53,71 @@ export function WalletDetailsHeader({
-            <WalletAvatar size="$8" wallet={wallet} />
-            {accountUtils.isHdWallet({ walletId: wallet?.id }) ? (
-              <ListItem.Avatar.CornerIcon
-                name="MenuCircleHorSolid"
-                color="$iconSubdued"
-              />
-            ) : null}
+    <YStack testID="account-selector-header" py="$1" {...rest}>
+      <ListItem gap="$1.5">
+        <XStack gap="$1.5" alignItems="center" flex={1}>
+          <Stack
+            role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)